### PR TITLE
chore: migrate to mage-tools 0.44

### PR DESCRIPTION
**Pull request by sousandrei**
Small migration to mage-tools 0.44﻿

### CODE REVIEW

File: .mage/go.mod [hunk 1]
```diff
@@ -3,12 +3,12 @@ module mage-tools
 go 1.17
 
 require (
+	github.com/go-logr/logr v1.2.2
 	github.com/magefile/mage v1.12.1
-	go.einride.tech/mage-tools v0.28.0
+	go.einride.tech/mage-tools v0.44.0
```
Comment (Edholm) [inline .mage/go.mod:8]: Already outdated :grinning:

**Review by Edholm — APPROVED** [review]
:+1:

File: .mage/go.mod (hunk 1 above)
Comment (sousandrei) [reply .mage/go.mod:8]: AAAA

Comment (Edholm) [reply .mage/go.mod:8]: rip auto-merge

**Pull request merged**